pkg/services/ngalert/metrics/historian.go

```diff
@@ -7,17 +7,59 @@ import (
 )
 
 type Historian struct {
-	WriteDuration *instrument.HistogramCollector
+	Info              *prometheus.GaugeVec
+	TransitionsTotal  *prometheus.CounterVec
+	TransitionsFailed *prometheus.CounterVec
+	WritesTotal       *prometheus.CounterVec
+	WritesFailed      *prometheus.CounterVec
+	WriteDuration     *instrument.HistogramCollector
+	BytesWritten      prometheus.Counter
 }
 
 func NewHistorianMetrics(r prometheus.Registerer) *Historian {
 	return &Historian{
+		Info: promauto.With(r).NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: Namespace,
+			Subsystem: Subsystem,
+			Name:      "state_history_info",
+			Help:      "Information about the state history backend.",
```

Comment:
nit: You use the word `store` in some helps but here you use backend.

Reply:
Moved everything -> store